ssl/t1_trce.c

```diff
@@ -557,6 +557,8 @@ static const ssl_trace_tbl ssl_sigalg_tbl[] = {
     {TLSEXT_SIGALG_rsa_pss_rsae_sha384, "rsa_pss_rsae_sha384"},
     {TLSEXT_SIGALG_rsa_pss_rsae_sha512, "rsa_pss_rsae_sha512"},
     {TLSEXT_SIGALG_rsa_pss_pss_sha256, "rsa_pss_pss_sha256"},
+    {TLSEXT_SIGALG_gostr34102012_256_intrinsic, "gost2012_256"},
```

Comment:
The same nit as above. Please put them along the other gost sigalgs.

Reply:
Ok, no problem i will but them along the gost in both places